packages/perseus/src/widgets/interactive-graphs/axis-labels.tsx

```diff
@@ -0,0 +1,61 @@
+import {vec} from "mafs";
+import React from "react";
+
+import {getDependencies} from "../../dependencies";
+
+import {pointToPixel} from "./graphs/use-transform";
+import useGraphConfig from "./reducer/use-graph-config";
+
+export default function AxisLabels() {
+    const {range, labels, width, height} = useGraphConfig();
+
+    const yAxisLabelLocation: vec.Vector2 = [0, range[1][1]];
+    const xAxisLabelLocation: vec.Vector2 = [range[0][1], 0];
+
+    const [xAxisLabelText, yAxisLabelText] = labels;
+    const graphInfo = {
+        range,
+        width,
+        height,
+    };
+    const [x1, y1] = vec.add(
+        pointToPixel(xAxisLabelLocation, graphInfo),
+        [0, 0],
+    );
```

Comment:
Not a big deal, but... this `vec.add()` call has no effect, since it's adding `[0, 0]`.